long-live-pomf.md

```diff
@@ -3,6 +3,7 @@
 | https://mixtape.moe/       | Up               | https://gitgud.io/Sapphire/mixtape.moe        |              100 | Pastebin, voice
 | https://safe.moe/          | Up               | https://github.com/WeebDev/loli-safe          |              200 | Pastebin
 | https://sub.god.jp/        | Up               | https://github.com/xatasan/registrars         |               32 | JS-free, pastebin
+| https://plexle.fi          | Up               | https://github.com/pomf/pomf                  |               20 | Zero Logs.               
```

Comment:
Looks like a pastebin site. Pretty sure it doesn't run `https://github.com/pomf/pomf`.

Reply:
Whoops sorry I meant to put https://l.shenron.su